tools/BUILD

```diff
@@ -3,3 +3,23 @@ licenses(["notice"])  # Apache 2
 exports_files([
     "check_envoy_includes.py",
 ])
+
+py_library(
+    name = "tools_lib",
+    srcs = [
```

Comment:
Prefer to just inline this in the target below, these intermediate targets aren't that helpful..

Reply:
Sure, will do

Reply:
Done in https://github.com/envoyproxy/nighthawk/commit/b1e3ad8e779f4ad0cac5498ead7e4d2e30778fb9